europython2024/README.md

```diff
@@ -0,0 +1,14 @@
+# No more spaghetti code: build reusable data pipeline with Sciline
```

Comment:
# Sciline: Assemble tasks graphs using type-hint-driven dependency injection

Reply:
I wanted a little bit of click-bait...

Reply:
Think about the audience: task graphs, type hints, and dependency injection *is* clickbait ;)